src/ast/mod.rs

```diff
@@ -2243,20 +2243,27 @@ pub struct Function {
     pub over: Option<WindowSpec>,
     // aggregate functions may specify eg `COUNT(DISTINCT x)`
     pub distinct: bool,
+    pub special: bool,
```

Comment:
I wonder why a `special` field is needed ? Wouldn't it be enough to match on the `name`? 

If we do need a `special` field, can you please add a `///` comment explaining what it means and what it is used for?

Reply:
I use this field to skip formating `(arguments)`.

<img width="294" alt="image" src="https://user-images.githubusercontent.com/572096/184127721-ef838706-2094-45f0-a69e-3e7d08efead0.png">

Right now, Dialect is not available in formating, because we use `fmt::Display` for implementing `to_string()` method.